### PR TITLE
Add MusicPlayer class component.

### PR DESCRIPTION
**Pull request by ringo196**
@kristie-zheng please check! Thanks.

### CODE REVIEW

File: client/src/components/musicPlayer.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import $ from 'jquery';
+import songs from '../../sampleData.js'
+
+class VideoPlayer extends React.Component {
```
Comment (kristiedesiree) [inline client/src/components/musicPlayer.jsx]: Probably want to rename your component to MusicPlayer

File: client/src/components/musicPlayer.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+	albumClickHandler() {
+		// goes to a static album page
+	}
+
+	genreClickHandler() {
+		// goes to a static genre page
+	}
+
+	albumArtClickHandler() {
```
Comment (kristiedesiree) [inline client/src/components/musicPlayer.jsx]: What exactly do you mean by "pops up" isn't it always showing

File: client/src/components/musicPlayer.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+			currentSong: 1
+		}
+	}
+
+	playButtonHandler() {
+		// plays a static song
+	}
+
+
+	artistClickHandler() {
```
Comment (kristiedesiree) [inline client/src/components/musicPlayer.jsx]: might not want to make links at all. maybe just have the link change upon hover/click

**Review by kristiedesiree — CHANGES_REQUESTED** [review]

**Comment by ringo196** [comment]
@kristie-zheng  Can you check again? Should be fixed now, also added a jquery dependency installation too.

**Review by kristiedesiree — APPROVED** [review]
Looks good; please put node_modules in git ignore

**Pull request merged**